app/common/gristUrls.ts

```diff
@@ -1318,3 +1294,19 @@ export interface UrlTweaks {
     url: URL,
   }): void;
 }
+
+function withAdminDefinedUrls(defaultUrls: ICommonUrls): ICommonUrls {
+  const adminDefinedUrlsStr = getCustomizableValue('adminDefinedUrls', "GRIST_CUSTOM_COMMON_URLS");
+  if (!adminDefinedUrlsStr) {
+    return defaultUrls;
+  }
+
+  const adminDefinedUrls = JSON.parse(adminDefinedUrlsStr);
```

Comment:
what happens if the `adminDefinedUrlStr` JSON parsing fails?

Reply:
It should crash and the server won't start.

Probably worth to add a meaningful error message for this case.

Reply:
Done